sdk/remoterendering/mixedreality-remoterendering/swagger/README.md

```diff
@@ -0,0 +1,210 @@
+# Azure Example TypeScript Protocol Layer
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+v3: true
+package-name: "@azure/mixedreality-remoterendering"
+title: RemoteRenderingRestClient
+description: Azure Remote Rendering Client
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../
+source-code-folder-path: ./src/generated
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/2a65b0a2bbd9113b91c889f187d8778c2725c0b9/specification/mixedreality/data-plane/Microsoft.MixedReality/stable/2021-01-01/mr-arr.json
+add-credentials: false
+package-version: 1.0.0-beta.1
+disable-async-iterators: true
+hide-clients: true
+use-core-v2: true
+use-extension:
+  "@autorest/typescript": "https://aka.ms/autorest/typescript/corev2"
```

Comment:
You could just do this now:
```suggestion
typescript: true
```

Reply:
@MalcolmTyrrell  This suggestion was not applied for some reason, was it by mistake?

Reply:
Sorry, I should not have closed this comment without a remark. I tried this, and the effect is that it ignores all the customizations, so classes have the wrong names.

Reply:
@sarangan12 could you please look into this? this change should just make autorest use the latest TS extension instead of the one in https://aka.ms/autorest/typescript/corev2.

Reply:
Actually, it looks like my autorest was out-of-date. With your change, I'm not getting reasonable code again. I see there are many changes though.

Reply:
@MalcolmTyrrell very weird because we have been using that latest version for a while now but may be there are edge cases in AAR. Could you please post an example? or have a draft PR with the regeneration so we can inspect the changes between it and here?

Reply:
Good news. This change works, and fixes some of the linter issues (it introduced the interfaces I was missing).